std/src/main/scala/algebra/std/byte.scala

```diff
@@ -0,0 +1,58 @@
+package algebra
+package std
+
+import algebra.lattice._
+import algebra.number._
+import algebra.ring._
+import algebra.std.util.StaticMethods
+
+package object byte extends ByteInstances
+
+trait ByteInstances {
+  implicit val byteAlgebra = new ByteAlgebra
+
+  val ByteMinMaxLattice = new Lattice[Byte] {
```

Comment:
isn't this really just the `OrderLattice[Byte]` or something? Why define this for each type with on Order?